test/locks.js

```diff
@@ -39,231 +47,219 @@ describe('Locks', function() {
           extend: 'apostrophe-locks',
           alias: 'locks3'
         }
-      },
-      afterInit: function(callback) {
-        assert(apos.modules['apostrophe-locks']);
-        assert(apos.modules['apostrophe-locks-1']);
-        assert(apos.modules['apostrophe-locks-2']);
-        assert(apos.modules['apostrophe-locks-3']);
-
-        // In tests this will be the name of the test file,
-        // so override that in order to get apostrophe to
-        // listen normally and not try to run a task. -Tom
-        apos.argv._ = [];
-        return callback(null);
-      },
-      afterListen: function(err) {
-        assert(!err);
-        done();
       }
     });
-  });
 
-  it('cleanup', function(done) {
-    apos.locks.db.remove({}, function(err) {
-      assert(!err);
-      done();
-    });
+    assert(apos.modules['apostrophe-locks']);
+    assert(apos.modules['apostrophe-locks-1']);
+    assert(apos.modules['apostrophe-locks-2']);
+    assert(apos.modules['apostrophe-locks-3']);
   });
 
-  it('should allow a single lock without contention uneventfully', function(done) {
-    let locks = apos.modules['apostrophe-locks'];
-    return async.series([ lock, unlock ], function(err) {
-      assert(!err);
-      done();
-    });
-    function lock(callback) {
-      return locks.lock('test', callback);
-    }
-    function unlock(callback) {
-      return locks.unlock('test', callback);
+  it('cleanup', async function() {
+    try {
+      const response = await apos.locks.db.remove({});
+
+      assert(response.result.ok === 1);
+    } catch (e) {
+      assert(false);
```

Comment:
try/catch not necessary because mocha will catch and treat as an assert failure.